english/music-festival/woodstock-1994/woodstock-1994.md

```diff
@@ -0,0 +1,10 @@
+You're trapped in a memory of Woodstock '94.  Were these your experiences,
+or do they belong to someone else... perhaps [many others](https://www.youtube.com/watch?v=Cz227EU_1mU)?
+
+You slide in the mud while being entertained by:
+[Blind Melon](https://www.youtube.com/watch?v=3xph8pPJMB8)
+[Primus](https://www.youtube.com/watch?v=v-HUK8qR9QA)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.